collections/_infographics/climate/map-temperature-change.md

```diff
@@ -3,14 +3,29 @@ layout:        infographic
 title:         "Map of temperature changes (1961–2019)"
 slug:          "map-temperature-change"
 redirect_from: "/map-temperature-change"
-published:     2022-07-26
+published:     2022-11-20
 weight:        99
 tags-scopes:   [ world ]
 tags-topics:   [ climate ]
-caption:       "The global temperature rise since 1961 is approximately 1.1°C. However, the warming rate is not the same for all locations: while the temperature in most of the planet's oceans has not increased by more than 0.8°C, most of the world's land is warming considerably faster. The highest warming rate on Earth (over 4°C in the last 60 years) is in the Arctic Ocean."
+caption:       "The global temperature rise since 1961 is approximately 1.1 °C. However, the warming rate is not the same for all locations: while the temperature in most of the planet's oceans has not increased by more than 0.8 °C, most of the world's land is warming considerably faster. The highest warming rate on Earth (over 4 °C in the last 60 years) is in the Arctic Ocean."
```

Comment:
switch "rise" to "increase" (cleaner, consistent with most-used term)

Reply:
I updated the caption. "rise" has been changed to "increased".